website/layouts/partials/footer.html

```diff
@@ -0,0 +1,44 @@
+{{ $links := .Site.Params.links }}
+<footer class="bg-dark py-5 row d-print-none">
+  <div class="container-fluid mx-sm-5">
+    <div class="row">
+      <p class="mx-auto col-sm-3 text-center text-light">
+        Built with ❤️ at <a class="text-light" href="https://aws.amazon.com/">AWS</a>
```

Comment:
Could we move this from a universal footer to an "about us" type page? where that page would include info on contributing to the project?